### PR TITLE
Release 1.5.1

### PR DESCRIPTION
**Pull request by julissa**
= 1.5.1 =
* Tested up to Wordpress 5.9.3
* Updates dependencies

### CODE REVIEW

**Review by antauth — APPROVED** [review]

**Review by eudaimonious — APPROVED** [review]

**Review by antauth — CHANGES_REQUESTED** [review]
Darian says we need to make a change to the README

**Comment by julissa** [comment]
> Darian says we need to make a change to the README

What's the change we should be making?

**Review by antauth — CHANGES_REQUESTED** [review]
Security has requested updates to the README to reflect how to manually create a GPG-signed version of the plugin.

Below is my suggested copy which should be part of the *Releasing a new version* section of the README.

4. Create a distributable zip archive of the plugin for upload as a Github release. The script does require that `GPG_PRIVATE_KEY` and `GPG_UID` are set as environment variables on your computer.

`./bin/archive.sh`

Upload the generated files (i.e. the .zip and the .zip.asc) to the correct release under https://github.com/actblue/actblue_wordpress_plugin/releases.

File: README.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ gpg:                using RSA key 44CD70EA353C06DE5AB608E4D842AF7166B99EDA
 gpg:                issuer "integrations@actbluetech.com"
 gpg: Good signature from "integrations@actbluetech.com" [full]
 ```
+4. Create a distributable zip archive of the plugin for upload as a Github release. The script does require that `GPG_PRIVATE_KEY` and `GPG_UID` are set as environment variables on your computer.
```
Comment (dap) [inline README.md:29]: This step should be in the Installation section further below in this file.

Comment (antauth) [reply README.md:29]: I don't believe it should fall there. Those instructions depend on running CirlceCI and it is a step in CircleCI that performs the manual steps outlined here.

**Comment by julissa** [comment]
> > Darian suggested that we add some info to the README and the WordPress plugin page about an expected release cadence for the plugin to make sure potential users don’t feel it's unmaintained or abandoned.
> > I propose we add in a **"Releases & Maintenance"** section and under that add in something like
> > _This plugin will be tested and released quarterly unless security patches or updates are needed in the intervening time._ or something like this. I can connect with Jax on copy if you'd like.

This copy is okay. I'd just like confirmation that it's final copy so I can update the README.

**Comment by AshlenPrice** [comment]
> 
Great! Lets make this final then and update the README

File: README.md (hunk 1 above)
Comment (dap) [reply README.md:33]: It should definitely be in the "Releasing a new version" section and it should definitely not be in the "Verifying GitHub Releases" section.

The "Verifying GitHub Releases" section documents the steps _a WordPress administrator_ can take to verify the signature of a released version of the plugin to ensure that the package has not been modified to effect a supply chain attack.

The "Releasing a new version" section documents the steps _an ATS developer_ takes in order to create and new distributable version of the plugin and tag in Github.

The content in question here, which documents signing a new distributable version of the plugin (i.e., the zip file), is, by definition, a release step and is _action taken by an ATS developer_, not a WordPress administrator.

**Review by dap — CHANGES_REQUESTED** [review]
Adjustment to README is still needed

File: README.md (hunk 1 above)
Comment (antauth) [reply README.md:33]: My bad, I thought it was there as that was where I asked for it to be placed. @julissa it does seem to be in the wrong section.

**Comment by julissa** [comment]
> Adjustment to README is still needed

@dap my mistake! updated.

**Comment by dap** [comment]
> > Adjustment to README is still needed
> 
> @dap my mistake! updated.

Thanks @julissa! No prob!

**Comment by dap** [comment]
What's the correct way to test the functionality of the plugin? I'm observing that it isn't working in the local environment that results from following the build steps. When I paste in a link to a donation form, I get the following:

image

**Comment by samuelcole** [comment]
Try an embeddable form: https://secure.actblue.com/donate/actblue-1-embed

**Comment by dap** [comment]
> Try an embeddable form: https://secure.actblue.com/donate/actblue-1-embed

That was it for the embed in the editor. Thanks @samuelcole!

**Comment by julissa** [comment]
> bin/run-e2e-tests.sh

What's the error you are getting? Are you running `bin/run-e2e-tests.sh` from the root?

**Comment by dap** [comment]
> > bin/run-e2e-tests.sh
> 
> What's the error you are getting? Are you running `bin/run-e2e-tests.sh` from the root?

@julissa, here's a gist with output. I am running it from the root of the project: https://gist.github.com/dap/3b561f95a54824d5cada18ffa40e93cd

**Comment by julissa** [comment]
Closing PR to update version number issue first.

**Pull request closed without merging**